### DISCUSSION TITLE
spinning wheeel

### DISCUSSION THREAD

**Discussion by snhnic** [music-assistant/support#3267] [Q&A] [answered]
How log does it take to index and how much storage does the index take? I have installe Music Assistant and put in the nas location. But now for two days iI see a spinning wheel inde right top saying "the server is processing a request or syncing music". I got music mp3 about 1 terra byte at least.

**Comment by OzGav** [comment, marked as answer]
Impossible to say as it depends also on the speed of the host running the server but a TB of MP3 I could only guess a week+

**Reply by snhnic** [reply]
Thanks, now I know it is not hanging. Wat about the size the index will take?

**Reply by OzGav** [reply]
Couldn't say as I know very little about your setup.
